gitops/argocd/tekton-chains/overlays/openshift/security-context-fix.yaml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
Any reason to place this one and chains-secrets-config-patch.yaml under overlays and not under base?

Reply:
The default configuration is expected to work fine on kubernetes. This patch is only required when deploying on OpenShift.